tests/unit/moduleapi/basics.tcl

```diff
@@ -44,3 +44,29 @@ start_server {tags {"modules external:skip"} overrides {enable-module-command no
        assert_error "ERR *MODULE command not allowed*" {r module load $testmodule}
     }
 }
+
+start_server {tags {"modules external:skip"} overrides {enable-debug-command no}} {
+    r module load $testmodule
+
+    test {debug command disabled} {
+        assert_equal {no} [r test.candebug]
+    }
+}
+
+start_server {tags {"modules external:skip"} overrides {enable-debug-command yes}} {
+    r module load $testmodule
+
+    test {debug command enabled} {
+        assert_equal {yes} [r test.candebug]
+    }
+}
+
+start_server {tags {"modules external:skip"} overrides {enable-debug-command local}} {
+    r module load $testmodule
+
+    test {debug commands are enabled for local connection} {
+        assert_equal {yes} [r test.candebug]
+    }
+
+    # TODO: Non-local case - can we test here?
```

Comment:
we can probably look up the local ip address and open a connection to it, but if we don't have another test that's already doing that, maybe we can skip it.
but if we do, let's test it once manually

Reply:
Tested locally, looks good:
![image](https://github.com/user-attachments/assets/eef689a6-609a-459c-8a63-ba3d05b0c3a3)